README.md

```diff
@@ -208,6 +208,10 @@ Makes Pieman restrict itself to creating only a chroot environment based on the
 
 Specifies the time zone of the system.
 
+##### CUSTOM_FILES=""
+
+Allows specifying the directory which will recursively copied to the chroot environment. All files and directories from passed folder will be copied relatively of top-most chroot directory.
+
```

Comment:
I've got a few remarks:
* The term `directory` is closer to the the unix-like operating systems than `folder`, so personally, I would choose the first one.
* I see some grammar problems with the description.
* I guess the second sentence is redundant.

Please, consider this variation on the description:
> Allows specifying the directory which will be recursively copied to the chroot environment.